### ISSUE TITLE
X connection error Raze and Gzdoom in the last version of Manjaro

### ISSUE THREAD

**Issue by orderorder** [coelckers/Raze#354]
```
ST_Init: Init startup screen.
Checking cmd-line parameters...
S_InitData: Load sound definitions.
G_ParseMapInfo: Load map definitions.
Texman.Init: Init texture manager.
ParseTeamInfo: Load team definitions.
LoadActors: Load actor definitions.
script parsing took 212.25 ms
R_Init: Init Doom refresh subsystem.
DecalLibrary: Load decals.
M_Init: Init menus.
P_Init: Init Playloop state.
ParseSBarInfo: Loading custom status bar definition.
D_CheckNetGame: Checking network game status.
player 1 of 1 (1 nodes)
Using video driver x11
X connection to :0 broken (explicit kill or server shutdown).
```

I try other launchers and games but seems a Gzdoom and Raze error.

**Comment by arrowgent** [comment]
what are some details?

version of raze (latest v1.0.3)
version of manjaro? (lsb_release -a)
kernel version (uname -a)
cpu
gpu & driver version (nvidia-smi)

glxinfo
do you have opengl?
vulkaninfo
do you have vulkan?


```
Using video driver x11
Vulkan device: GeForce GTX 1070
Vulkan device type: discrete gpu
Vulkan version: 1.2.133 (api) 450.476.192 (driver)
```

suggest to me the very next section after that is requesting to access OPENGL (default) or VULKAN (custom setting) output

**Comment by alexey-lysiuk** [comment]
"Last version master branch" is completely meaningless description. If you’re are using development version, post its commit hash.

Does Raze 1.0.3 work fine?

**Comment by sinisterseed** [comment]
Plus, the fact that it worked fine before seems to point to a distro issue.

Does 1.0.3 still work for you?

**Comment by orderorder** [comment]
i have compiled raze again (development version commint d466ec0). I have noticed that it is randomly able to run some of the supported games, but on retry it shows the same error. Version 1.0.3 does not work, same error. I have tried the following launchers:

https://github.com/yquake2/yquake2
https://github.com/svkaiser/strife-ve
https://github.com/JACoders/OpenJK

They work. Updates don't seem to have affected them. I'm going to keep trying things to see if I draw any conclusions.

**Comment by Cacodemon345** [comment]
Did you experience this issue with other SDL2 games? Because this looks like a SDL2 bug.

**Comment by sinisterseed** [comment]
The other ports you tried also do not tell us much, one of them is even a Quake 2 port and the other is engine improvements for Jedi Academy plus the Strife remaster, while those are Build engine games.

**Comment by orderorder** [comment]
Yes other SDL2 based port work. Do you want me to try a particular SDL2 port?

**Comment by Cacodemon345** [comment]
What DE do you use?

**Comment by orderorder** [comment]
KDE

**Comment by orderorder** [comment]
as the gzdoom gives me the same problem I have tried the prboom-plus. But this one does work

**Comment by orderorder** [comment]
I have compiled SDL2 but still the same error

**Comment by orderorder** [comment]
In another pc with the lastest version of Manjaro, Mesa Drivers and a RX580, Raze works well

**Comment by Cacodemon345** [comment]
Looks like a hardware bug specific to your AMD APU system.

**Comment by orderorder** [comment]
I have been able to verify that it may be a problem with my configuration in KDE. Raze works fine in Enlightenment. On my other pc with the RX580 I use Gnome.

**Comment by orderorder** [comment]
This thread may have to be closed, since I can't find the fault and it may be a matter of my configuration. Can anyone confirm that the same thing happens with KDE?

**Comment by Cacodemon345** [comment]
https://gitlab.freedesktop.org/mesa/mesa/-/issues/4763

Turns out it's a Mesa bug.
